packages/auth/src/mfa/mfa_session.ts

```diff
@@ -31,11 +32,19 @@ interface SerializedMultiFactorSession {
 export class MultiFactorSessionImpl implements MultiFactorSession {
   private constructor(
     readonly type: MultiFactorSessionType,
-    readonly credential: string
+    readonly credential: string,
+    readonly auth?: AuthInternal
```

Comment:
Wondering if this change need to be documented somewhere or any proto files need to be changed as we added a new param

Reply:
This is an implementation detail, a developer will not directly interact with this field, hence it is not documented.

Reply:
What about internal customers?